net/isc-dhcp/files/dhcpd.init

```diff
@@ -344,6 +344,11 @@ static_hosts() {
 	config_foreach static_host_add host "$@"
 }
 
+eval_ipcalc() {
```

Comment:
Why is this necessary?  Also, see PR #21410

Reply:
This change is coming from https://github.com/openwrt/openwrt/pull/12925 and is only slightly related.
Instead of enabling modes of configuration, it is concerned with error handling.

The synopsis is: ipcalc.sh now fails if the range doesn't fit into the subnet.
The purpose of the function is to pass through the return code when using `eval`. While we're at it, don't call eval if ipcalc.sh fails.
A similar patch is pending for dnsmasq.

Reply:
Hmm... why not wrap it with `export -n "$(...)"` instead?

Reply:
Hmm, the quotes won't work. `IP` would be set to the entire output of `ipcalc.sh`.
xargs has no `-d` in busybox, also xargs won't work with the export built-in.

A more correct solution would be `while read LINE ; do export -n "$LINE"; done < <(ipcalc.sh 11.53.100.0 30 1 2)`.
If you want, you can grap that and pour it into a commit.
`ipcalc.sh 11.53.100.0 30 1 2 | while read LINE ; do export -n "$LINE"; done` doesn't work because the `export` runs in a subshell (because of the pipe).

Tbh, I don't think the added safety is worth the added complexity. The output of ipcalc.sh is intended for something like `eval`.